app/Resources/DoctrineMigrations/Version20150813184053.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Ilios\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
```

Comment:
Please add a description of the migration in the comment and remove the auto generated junk.